package.json

```diff
@@ -99,9 +100,8 @@
     "glob": "^10.3.3",
     "husky": "^8.0.3",
     "jest": "29.7.0",
-    "jest-diff": "^29.6.2",
-    "jest-snapshot": "^29.6.2",
     "jest-specific-snapshot": "^8.0.0",
+    "knip": "^4.0.2",
```

Comment:
> Knip has a lot of dependencies (200+) ([npmgraph](https://npmgraph.js.org/?q=knip)). Maybe we should only install it in CI?

Personally, I don't mind. The lockfile is pretty huge and uses nx/Yarn caching anyway. If we want to make the story for large devDependencies better we'd probably want to investigate a more holistic strategy. But I won't hold this opinion strongly if anybody has a strong preference.

FWIW a `yarn dedupe` does remove a few entries... not too many though.

Reply:
Makes sense! Thanks!